src/Features/SupportDataBinding/BrowserTest.php

```diff
@@ -130,4 +131,43 @@ public function render(): string
             ->waitForLivewire()->select('@parent', 'foo')
             ->assertSelected('@child', 'bar');
     }
+
+
+    /** @test */
+    public function it_can_update_checkbox_with_value()
+    {
+        // Check if checking a checkbox results in correct value
+        Livewire::visit(new class() extends \Livewire\Component
+        {
+            public $element = '';
+
+            public function render(): string
+            {
+                return <<<'HTML'
+                <div>
+                    <input dusk="checkbox" type="checkbox" value="checked" wire:model.live="element">
+                    <span>{{ $element }}</span>
+                </div>
+                HTML;
+            }
+        })
+            ->check('@checkbox')
+            ->assertSee('checked');
+
+        // Check if checkbox is checked when value is the same as model value
+        Livewire::visit(new class() extends \Livewire\Component
+        {
+            public $element = 'checked';
+
+            public function render(): string
+            {
+                return <<<'HTML'
+                <div>
+                    <input dusk="checkbox" type="checkbox" value="checked" wire:model.live="element">
+                </div>
+                HTML;
```

Comment:
you forgot to display the property `<span>{{ $element }}</span>` 🙃

Reply:
It's not needed in the second test, as it tests if the checkbox itself is checked or not

Reply:
mmm... yeah it's using `->assertChecked('@checkbox')`
![image](https://github.com/livewire/livewire/assets/58886605/d663d6be-73c3-407e-9712-c3537344d3a7)